src/components/grid-aware/HomePageLargeParagraph/HomePageParagraph.module.css

```diff
@@ -0,0 +1,34 @@
+.text {
+  font: var(--font-headline);
+  text-align: center;
+}
+
+.bleedWrapper {
+  display: grid;
+  align-items: center;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
```

Comment:
Since you only have one grid row, I don't think it's necessary to define two grid rows, and you can even omit this property altogether because the default behavior is to have one auto-sized row.

Reply:
Deleted  "grid-template-rows: auto 0." That is the property you are talking about above correct?

Lines 7-9 are grid tracks defining the column which builds the bleed columns, so I keep those right?